stable/xray/values.yaml

```diff
@@ -450,10 +468,24 @@ server:
   replicaCount: 1
   internalPort: 8000
   externalPort: 80
+  annotations: {}
```

Comment:
@evannorstrand I'm really sorry for bugging you, but I see that you added the `customVolumes` and `customVolumeMounts` to `indexer` as well. Can you please add that to the README? Thanks again for your time and effort with this contribution!

Reply:
Once you add it to the README, I will approve and we can merge :)